src/error/metadata/codes/E032.md

```diff
@@ -0,0 +1,3 @@
+This error occurs when rover tries to start a graph or subgraph asynchronous check with invalid inputs.
+
+Please double check your inputs before running again.
```

Comment:
what is an example of an invalid input? what is valid input? how do i know what to put instead?

Reply:
++ still need this